Review edits (#1330)
hadley/r4ds: intro.qmd

First, you must **import** your data into R.
This typically means that you take data stored in a file, database, or web application programming interface (API) and load it into a data frame in R.

> If you can't get your data into R, you can't do data science on it with R!

Reviewer: ```suggestion
If you can't load your data, you can't do data science on it!
```

Underlying all these tools is **programming**.
Programming is a cross-cutting tool that you use in nearly every part of a data science project.
You don't need to be an expert programmer to be a successful data scientist, but learning more about programming pays off because becoming a better programmer allows you to automate common tasks and solve new problems with greater ease.

You'll use most of these tools in every data science project, but they're not enough for most projects.
In our experience, there's a rough 80-20 rule at play; you can tackle about 80% of every project using the tools you'll learn in this book, but you'll need other tools to tackle the remaining 20%.
Throughout this book, we'll point you to resources where you can learn more.

## How this book is organized

The previous description of the tools of data science is organized roughly according to the order in which you use them in an analysis (although, of course, you'll iterate through them multiple times).
In our experience, however, learning data ingesting and tidying first is sub-optimal because in many cases it's a fairly easy task, and we don't want to get bogged down in the complicated cases for now.

> Instead, we'll start with imported and tidied data, so we can go straight to making interesting insights through visualization and transformation.

Reviewer: ```suggestion
Instead, we'll start with pre-loaded tidy data, so we can go straight to making interesting insights through visualization and transformation.
```
